src/Bonsai.Gui.ZedGraph/BoundedGraphPanel.cs

```diff
@@ -41,50 +41,53 @@ public int Capacity
 
         protected override void OnInvalidated(InvalidateEventArgs e)
         {
-            double? maxValue = null;
-            var curveList = GraphPane.CurveList;
-            for (int i = 0; i < curveList.Count; i++)
+            if (!(GraphPane is null))
```

Comment:
```suggestion
            if (GraphPane == null)
            {
                return;
            }
```

Can we modify the test to break out of the function on `null`? This would reduce the differences overall in the file.